### PR TITLE
Wiki updated

### PR DESCRIPTION
**Pull request by neelimagoogly**
I have made the following changes to the docs of Wiki : 

1. Restructured all docs.
2. Fixed outdated images and added new images(and a video).
3. Added appropriate styles to text . 
4. Added links wherever necessary.
5. Added emojis :)

@robsavoye and @petya-kangalova please review it.

### CODE REVIEW

File: docs/About.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 # Field Mapping Tasking Manager (FMTM)
 
+## :open_book: History 
+_How was FMTM born ?_
+<br>
+<br>
+It started due to the efforts of Rob Savoye, Senior Technical Lead at Humanitarian OpenStreetMap Team. HOT uses ODK heavily, but most of the data never makes it into OSM because all the data processing is manual and slow, so it doesn't get done.<img align="right" width="300px" src="https://github.com/hotosm/osm-fieldwork/assets/97789856/afc791c7-6cc9-4496-aa93-ab02733f30b8"/> 
+Rob convinced HOT to let him work on this as a small side project. Then Ivan Gayton(Senior Humanitarian Advisor at Humanitarian OpenStreetMap Team) heard about what Rob was working on and goes "That's the missing piece I needed!". He'd been wanting to build FMTM for years, but lacked the ability to process the data. He got HOT interested enough to direct some resources to his idea, so FMTM was born.
```
Comment (petya-kangalova) [inline docs/About.md:10]: @neelimagoogly suggested edit for the history paragraph is  how we frame it.  It started as a collaborative project, explaining that both Rob and Ivan have been instrumental in brining the idea. A [webinar](https://www.youtube.com/watch?v=GiLKRZpbtrc&ab_channel=HumanitarianOpenStreetMapTeam) then took place in September 2022 that showcases the high interest from the community and the need for collaborative field mapping that really kicked off the starting point for building the Field Mapping Tasking Manager.

Comment (robsavoye) [reply docs/About.md:10]: Yes for sure. Although I was focused on the low level coding, FMTM itself is all Ivan's idea from the very beginning. He did the first FMTM webinar when he realized we could actually build his vision, but it was all his idea. All I did was integrate osm-fieldwork into a FastAPI backend. After the webinar Kristen built the frontend demo we did in Naivasha, which was responsible for where the project is now.

Comment (neelimagoogly) [reply docs/About.md]: I have made the changes suggested for the history  @robsavoye  and @petya-kangalova :)

**Comment by neelimagoogly** [comment]
I updated the Getting Started page in an organised way such that now this page now can be used as both a guide for users wrt issue #429 as well as guide for developers. @robsavoye and @petya-kangalova please have look into the updated **Getting Started page** and review it .

File: docs/About.md [hunk 2]
```diff
@@ -2,6 +2,19 @@
 
 # Field Mapping Tasking Manager (FMTM)
 
+## :open_book: History 
+_How was FMTM born ?_
+<br>
+<br>
+It started as Ivan's idea to build FMTM (Ivan Gayton is Senior Humanitarian Advisor at Humanitarian OpenStreetMap Team) which then became a collaborative project with the efforts of Ivan , Rob Savoye who is Senior Technical Lead at Humanitarian OpenStreetMap Team and many other members from HOT.
```
Comment (petya-kangalova) [inline docs/About.md:9]: @neelimagoogly  please add  after "many other members from HOT as well as volunteers interested in the project".

File: docs/About.md [hunk 3]
```diff
@@ -2,6 +2,19 @@
...
+HOT uses ODK heavily, but most of the data never makes it into OSM because all the data processing is manual and slow, so it doesn't get done.<img align="right" width="300px" src="https://github.com/hotosm/osm-fieldwork/assets/97789856/afc791c7-6cc9-4496-aa93-ab02733f30b8"/> 
+Rob convinced HOT to let him work on this as a small side project. Then Ivan Gayton(Senior Humanitarian Advisor at Humanitarian OpenStreetMap Team) heard about what Rob was working on and goes "That's the missing piece I needed!". He'd been wanting to build FMTM for years, but lacked the ability to process the data.A [webinar](https://www.youtube.com/watch?v=GiLKRZpbtrc&ab_channel=HumanitarianOpenStreetMapTeam) then took place in September 2022 that showcased the high interest from the community and the need for collaborative field mapping that really kicked off the starting point for building the Field Mapping Tasking Manager. It was Ivan who got HOT interested enough to direct some resources to his idea, so FMTM was born.
```
Comment (petya-kangalova) [inline docs/About.md:11]: @neelimagoogly  I would remove the first sentence here.  The rest looks great!

File: docs/About.md (hunk 2 above)
Comment (neelimagoogly) [reply docs/About.md:9]: Sure 👍

**Pull request merged**